stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/attachment/AttachmentViewHolder.kt

```diff
@@ -0,0 +1,37 @@
+package io.getstream.chat.android.ui.message.list.adapter.viewholder.attachment
+
+import android.content.Context
+import android.view.View
+import io.getstream.chat.android.client.models.Message
+
+/**
+ * An inner ViewHolder for custom attachments within message ViewHolder.
+ * Follows the lifecycle of the outer message ViewHolder.
+ *
+ * @param itemView The view that this ViewHolder controls.
+ */
+public abstract class AttachmentViewHolder(public val itemView: View) {
```

Comment:
Calling this a `ViewHolder` might be slightly confusing for users who see it and expect a `RecyclerView.ViewHolder`.